identity/client/src/pages/mappings/modals/AddModal.tsx

```diff
@@ -38,51 +45,61 @@ const AddMappingModal: FC<UseModalProps> = ({ open, onClose, onSuccess }) => {
     if (success) {
       enqueueNotification({
         kind: "success",
-        title: t("Mapping created"),
-        subtitle: t("You have successfully created mapping {{ name }}", {
-          name,
-        }),
+        title: t("mappingCreated"),
+        subtitle: t("mappingCreatedSuccessfully", { name: mappingName }),
       });
       onSuccess();
     }
   };
 
   return (
     <FormModal
-      headline={t("Create new mapping")}
+      headline={t("createNewMapping")}
       open={open}
       onClose={onClose}
       loading={loading}
       submitDisabled={submitDisabled}
-      confirmLabel={t("Create mapping")}
+      confirmLabel={t("createMapping")}
       onSubmit={handleSubmit}
     >
       <TextField
-        label={t("Mapping ID")}
-        placeholder={t("Enter mapping ID")}
+        label={t("mappingId")}
+        placeholder={t("enterMappingId")}
         onChange={setMappingId}
         value={mappingId}
+        helperText={t("uniqueIdForMapping")}
         autoFocus
       />
       <TextField
-        label={t("Mapping name")}
-        placeholder={t("Enter mapping name")}
+        label={t("mappingName")}
+        placeholder={t("enterMappingName")}
         onChange={setMappingName}
         value={mappingName}
+        helperText={t("uniqueNameForMapping")}
         autoFocus
```

Comment:
I think we can remove this line, as only one field should be autofocused once the modal is opened (the first one, which in this case is mappingId).